docs/sources/logql/metric_queries.md

```diff
@@ -0,0 +1,164 @@
+---
+title: Metric queries
+weight: 20
+---
+
+# Metric queries
+
+Metric queries extend log queries by applying a function to log query results.
+This powerful feature creates metrics from logs.
+
+Metric queries can be used to calculate the rate of error messages or the top N log sources with the greatest quantity of logs over the last 3 hours.
+
+Combined with parsers, metrics queries can also be used to calculate metrics from a sample value within the log line, such as latency or request size.
+All labels, including extracted ones, will be available for aggregations and generation of new series.
+
+## Range Vector aggregation
+
+LogQL shares the [range vector](https://prometheus.io/docs/prometheus/latest/querying/basics/#range-vector-selectors) concept of Prometheus.
+In Loki, the selected range of samples is a range of selected log or label values.
+
+The aggregation is applied over a time duration.
+Loki defines [Time Durations](https://prometheus.io/docs/prometheus/latest/querying/basics/#time-durations) with the same syntax as Prometheus.
+
+Loki supports two types of range vector aggregations: log range aggregations and unwrapped range aggregations.
```

Comment:
Having read this far, as a new user I would be a little lost. Is there something we can do with illustrative examples along the way to gradually build the mental model that folks need to understand our metric queries?